modules/data_release/README.md

```diff
@@ -1,16 +1,40 @@
-# Data Release Module
+# Data Release
 
-The Data Release Module can be used to easily distribute packaged data releases of your study.
+## Purpose
 
-## Features:
+The Data Release module can be used to easily distribute packaged data releases
+of the study.
```

Comment:
This sentence really feels like it needs another clause saying to whom it's releasing data and why you would want to use it over the alternatives

Reply:
OK. Tried to expand but the phrasing might need some improvement... (I'll commit everything once I have been through all your comments)